.github/workflows/basic.yml

```diff
@@ -208,11 +210,28 @@ jobs:
     - name: Checkout
       uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
 
+    - name: Get CMake for lukka/run-vcpkg
+      uses: lukka/get-cmake@latest
+      with:
+        cmakeVersion: 3.22.6
+
+    - name: Initialize vcpkg
+      uses: lukka/run-vcpkg@v11.5
+      with:
+        vcpkgGitCommitId: 3dd44b931481d7a8e9ba412621fa810232b66289
```

Comment:
how did you come up with this? is it the version of vcpkg available on GHA runners? or something else?

Reply:
@KFilipek how did you come up with this? ;-)

Reply:
As far as I remember, it's taken from their (vcpkg) repository.

Reply:
https://github.com/microsoft/vcpkg/commit/3dd44b931481d7a8e9ba412621fa810232b66289